### ISSUE TITLE
Unable to install (first time user)

### ISSUE THREAD

**Issue by nickreiner** [AmazMod/AmazMod#788]
I used the Windows installer. I'm trying to install AmazMod for the first time. The app is installed on the phone and the watch is connected to the PC. I run the installer, it installs the drivers, finds the device attached and starts the daemon.

After about 5 minutes I get an error `Failure [INSTALL_FAILED_DEXOPT]`

I have 1GB+ free space on the watch.

https://i.imgur.com/PlUH2KA.png

I have a Pace.

**Comment by fcaronte** [comment]
> I used the Windows installer. I'm trying to install AmazMod for the first time. The app is installed on the phone and the watch is connected to the PC. I run the installer, it installs the drivers, finds the device attached and starts the daemon.
> 
> After about 5 minutes I get an error `Failure [INSTALL_FAILED_DEXOPT]`
> 
> I have 1GB+ free space on the watch.
> 
> https://i.imgur.com/PlUH2KA.png
> 
> I have a Pace.

About the error maybe a watch reboot will fix, but if you see the service on watch try to reboot the phone and try again

**Comment by nickreiner** [comment]
I tried rebooting both the phone and the watch several times and the same error appeared.

However, I managed to install the service by pressing CTRL + C after the AMS Will Be Installed message and I got a "Success" message after that. Somehow I got the service installed on the watch. It seems like it works fine. Is there a way to check if AmazMod is properly working? I can access the widget, I can send files from my phone via the File Manager
